OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix for reports component Second to Last Phase

# Description
Lint fix for Reports component, this is phase 13 （Second to Last Phase）.

## Files changed
src/components/Reports/PeopleReport/PeopleReport.jsx

eslint test:
![CleanShot 2023-10-05 at 02 56 22@2x](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/64fb9406-211d-4a70-8884-bb98ef312c11)




# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint src/components/Reports/PeopleReport/PeopleReport.jsx`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!
